ast.go

```diff
@@ -1098,16 +1150,21 @@ func (node *ColumnConstraintPrimaryKey) String() string {
 		constraintName = fmt.Sprintf("constraint %s ", node.Name.String())
 	}
 
-	if node.Order == ColumnConstraintPrimaryKeyOrderEmpty {
-		return fmt.Sprintf("%sprimary key", constraintName)
+	var autoIncrement string
+	if node.AutoIncrement {
+		autoIncrement = " autoincrement"
+	}
+
+	if node.Order == PrimaryKeyOrderEmpty {
+		return fmt.Sprintf("%sprimary key%s", constraintName, autoIncrement)
 	}
-	return fmt.Sprintf("%sprimary key %s", constraintName, node.Order)
+	return fmt.Sprintf("%sprimary key %s%s", constraintName, node.Order, autoIncrement)
 }
 
 const (
-	ColumnConstraintPrimaryKeyOrderEmpty = ""
```

Comment:
This is a small change in nomenclature because the table constraints started using these variables. So I removed the `ColumnConstraint` because it does not only apply to column constraint.